.travis.yml

```diff
@@ -13,6 +13,15 @@ before_install:
   - |
     # install general dependencies used by more than one job
     set -eo pipefail
+    
+    if [ "$TEST_HARNESS" == true ]; then
+      sudo apt-get -qq -y install cmake libgtest-dev xvfb wmctrl xdotool xfwm4;
+      pushd /usr/src/gtest;
+      sudo cmake CMakeLists.txt;
+      sudo make;
+      sudo cp *.a /usr/lib
+      popd
+    fi;
```

Comment:
Please move this inside the `"$TRAVIS_OS_NAME" == "linux"` check right below it since it won't ever do anything on Mac, for consistency. Move it right after the check for `"$AUDIO" == "OpenAL"`.

Reply:
I'm guessing this is resolved, but if not, don't worry about it too much. We want a Windows implementation of TestHarness, as well.

Reply:
I got to it, but no, Travis will not do Windows, unless you mean test the harness with MinGW, in which case the OS will still be Linux. So it's good now!